src/main/java/food/finder/foodfinder/FoodFinderApplication.java

```diff
@@ -10,4 +10,4 @@
 	public static void main(String[] args) {
 		SpringApplication.run(FoodFinderApplication.class, args);
 	}
-}
\ No newline at end of file
+}
```

Comment:
Why did this pick up on the diff? Whitespace?

Reply:
I was confused at that too